Examples/DpdkExample-FilterTraffic/main.cpp

```diff
@@ -379,20 +379,22 @@ int main(int argc, char* argv[])
 			}
 			case 'p':
 			{
-				srcPortToMatch = atoi(optarg);
-				if (srcPortToMatch <= 0)
+				int ret = atoi(optarg);
+				if (ret <= 0)
```

Comment:
Also check the max valid port 

```suggestion
				if (ret <= 0 || ret > 65535)
```